jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryAuthenticationFailedException.java

```diff
@@ -20,12 +20,28 @@
 public class RegistryAuthenticationFailedException extends Exception {
 
   private static final String REASON_PREFIX = "Failed to authenticate with the registry because: ";
```

Comment:
Should we include the `registry` and `imageName` in this message?

Reply:
[BuildStepsRunner turns this into a `RegistryUnauthorizedException`](https://github.com/GoogleContainerTools/jib/blob/master/jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/BuildStepsRunner.java#L203) and throws away the `RegistryAuthenticationFailedException` referencing the underlying `HttpResponseException` instead.

Reply:
I had wondered if we could replace the use of `RegistryAuthenticationFailedException` with `RegistryUnauthorizedException`, but it does convey slightly different information.

I'll fix up the message.